adserver/templates/adserver/advertiser/overview.html

```diff
@@ -62,11 +62,51 @@ <h6 class="text-muted text-center">{% trans 'Spend' %}</h6>
     </div>
     <p class="text-right mb-5"><a href="{% url 'advertiser_report' advertiser.slug %}?start_date={{ start_date|date:'Y-m-d' }}&end_date={{ end_date|date:'Y-m-d' }}">{% trans 'detailed reporting' %} &raquo;</a></p>
 
-    <div class="row mb-5">
-      <div class="col min-vh-50">
-        {% metabase_question_embed metabase_advertiser_performance advertiser_slug=advertiser.slug start_date=start_date end_date=end_date %}
+    {% if advertiser_new %}
+      {# Advertiser onboarding workflow #}
+      <div class="row mb-5">
+        <div class="col">
+          <h3>{% trans 'There are a few steps to getting started with your first ad campaign with us.' %}</h3>
+          <ol class="mt-5 lead">
+            <li>
+              {% url "flight_list" advertiser.slug as flight_list_url %}
+              <p>{% blocktrans %}View the upcoming <a href="{{ flight_list_url }}">flights</a> set up for you and make sure the content and geographic targeting matches your customer focus. The price of your campaign depends on your chosen targeting.{% endblocktrans %}</p>
+            </li>
+            <li>
+              {% url "support" as support_url %}
+              <p>{% blocktrans %}Once your flights are set up, we will issue you an invoice. You can upload your ad creatives to your flights.{% endblocktrans %}</p>
```

Comment:
What is `set up`? Do they need to take an action here? Email us?

Reply:
Generally not. I'll clarify.